source/prerequisites/sql-server.txt

```diff
@@ -10,4 +10,117 @@ SQL Server Migration Prerequisites
    :depth: 1
    :class: singlecol
 
-SQL Server.
\ No newline at end of file
+Connecting Relational Migrator to SQL Server requires Change Data 
+Capture (CDC) mode enabled. When you set up a new connection, 
```

Comment:
I would prefer not to mention CDC in the first sentence. It will likely confuse and alarm people who are only running snapshot jobs - we can get to the detail later in the procedures.. Instead can we just say "To run migration jobs from a SQL Server source database, the database may require some configuration changes" (or similar).